src/components/LabelContentRow.tsx

```diff
@@ -15,7 +15,7 @@ const StyledLabel = styled(NormalText)`
   font-weight: normal;
 `;
 
-const StyledText = styled(NormalText)`
+const StyledText = styled.div`
   display: inline-block;
   font-weight: bold;
   flex: 1;
```

Comment:
siden vi vil unngå å bruke divs til tekst, så burde kanskje hele denne komponenten skrives om? Er children hva som helst? isåfall burde det renames fra StyledText til StyledContainer ellernoe. Tror klagingen på denne kommer fra feil bruk av komponenten eller?

Reply:
burde uansett fjerne `font-weight: bold` her

Reply:
Dette som fort blir litt kinkig med felleskomponenter... noen plasser er skal det bare være en tekst, mens andre ganger skal det være flere tekstkomponenter, men felles for alle er at de skal være bold. Navnet er uansett misvisende, holder det å bare rename til `StyledTextContainer`? @pcb-unit har kanskje tanker også?

Reply:
Vi får vel bare refraktorere etterhver som vi ser behovene endre seg. Splitte opp felleskomponenter eller slå de sammen alt etter som. Så spørs det også hvor mye vi skal satse på slike felles-komponenter når vi ikke vet hvordan stilingen skal være. Skermbildene med stiling er jo fremdeles bare skisser fra wimsical.

Reply:
ok, kanskje bare rename for nå da 👍